### PR TITLE
Add more I don't know responses to multiple choice questions, and allow up to two fixed responses to be anchored

### PR DESCRIPTION
**Pull request by moduloprime**
@adrifelt Added I don't knows, and made new constants.Randomize.ANCHOR_LAST_TWO so you can have both I don't know and Other anchored.

### CODE REVIEW

File: extension/surveys/survey-generator.js [hunk 1]
```diff
@@ -459,8 +459,11 @@ function getDomNameFromValue(answer) {
  * @returns {Array.Object} The randomized array.
  */
 function knuthShuffle(arr, randomType) {
-  var end = randomType == constants.Randomize.ANCHOR_LAST ?
-            arr.length - 1 : arr.length;
+  var end = arr.length;
+  if (randomType === constants.Randomize.ANCHOR_LAST)
+    end = arr.length - 1;
+  if (randomType === constants.Randomize.ANCHOR_LAST_TWO)
+    end = arr.length - 2;
```
Comment (adrifelt) [inline extension/surveys/survey-generator.js]: You probably also want to update flipArray, which also checks ANCHOR_LAST.

File: tools/processresults.py [hunk 2]
```diff
@@ -11,13 +11,18 @@
 import re
 
 DOGFOOD_START_DATE = datetime(2014, 12, 01, 0, 0, 0, 0)
+DEMOGRAPHIC_STABLE_DATE = datetime(2014, 12, 18, 0, 0, 0, 0)
```
Comment (adrifelt) [inline tools/processresults.py]: Did you mean for all of these python changes to be in the same PR?

**Comment by adrifelt** [comment]
Can you update the name of the PR to be more meaningful pls?

File: tools/processresults.py (hunk 2 above)
Comment (moduloprime) [inline tools/processresults.py]: No; I must've screwed up something with git

File: tools/processresults.py [hunk 3]
```diff
@@ -11,13 +11,18 @@
...
+DEMOGRAPHIC_CSV_PREFIX = 'demographics'
 CONDITIONS = [
     'ssl-overridable-proceed', 'ssl-overridable-noproceed',
     'ssl-nonoverridable', 'malware-proceed', 'malware-noproceed',
     'phishing-proceed', 'phishing-noproceed', 'extension-proceed',
     'extension-noproceed']
+TECHFAMILIAR_QUESTION_PREFIX = ('How familiar are you with each of the '
```
Comment (adrifelt) [inline tools/processresults.py]: This seems to be conceptually different from the rest of the PR which (by the description) is about adding IDKs

**Pull request closed without merging**